core/test/qa_YamlPmt.cpp

```diff
@@ -0,0 +1,722 @@
+#include "pmtv/pmt.hpp"
+#include <boost/ut.hpp>
+
+#include <cstdint>
+#include <fmt/format.h>
+#include <fmt/ranges.h>
+#include <fmt/std.h>
+
+#include <gnuradio-4.0/YamlPmt.hpp>
+#include <limits>
+#include <variant>
+
+template<typename T>
+std::string_view typeName() {
```

Comment:
We have a `gr::meta::type_name<T>()` definition that would provide a non-mangled name.